compose/cli/main.py

```diff
@@ -1160,8 +1160,9 @@ def remove_container(force=False):
                 )
                 pty = PseudoTerminal(project.client, operation)
                 sockets = pty.sockets()
```

Comment:
I could simplify the code as follows:

    pty.start(sockets)
    service.start_container(container)

but I was worried to reintroduce a bug during interactive run with networking, as described in f3e55568d1a7c111398876bec84da8ed8b42da7f